cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -608,9 +608,11 @@ public void changeFlowletStreamConnection(HttpRequest request, HttpResponder res
   @PUT
   @Path("/apps/{app-id}")
   public BodyConsumer deploy(HttpRequest request, HttpResponder responder, @PathParam("app-id") final String appId,
+                             @QueryParam(Constants.Scheduler.SUSPEND_SCHEDULES_DEPLOY_OPTION)
```

Comment:
we normally don't use constants for query parameters. It is easier to see what an actual request looks like if you put the literal string here. See other places in this class.